src/main/resources/templates/views/profile.html

```diff
@@ -7,6 +7,7 @@
     <body th:fragment="content">
         <div class="title text-center">
             <h1>Profile</h1>
+            <a href="create-sub-league.html"><input type="submit" class="btn btn-primary" style="float:right" value="Create Sub League"/></a>
```

Comment:
This should be calling a Controller action rather than access a html file directly